docs/Tutorial_Time_Series_Snippets.ipynb

```diff
@@ -1,8 +1,51 @@
 {
```

Comment:
<p>I don't think you need a header called "Snippets". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/397/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>